Rollbar/RollbarClient.cs

```diff
@@ -23,19 +27,22 @@ public RollbarClient(RollbarConfig config)
             Config = config;
         }
 
-        public RollbarResponse PostAsJson(Payload payload)
+        public RollbarResponse PostAsJson(Payload payload, IEnumerable<string> scrubFileds)
```

Comment:
should this be scrubFields? (same comment for the other two times `scrubFileds` is in this file)

Reply:
Ditto on the typos in the argument names, there are also similar typos in the tests `DataFiledFilteringTest` for example.

Reply:
fixed. thanks!